url-updater/src/main/java/com/devonfw/tools/ide/url/updater/kotlin/KotlinNativeCrawler.java

```diff
@@ -0,0 +1,37 @@
+package com.devonfw.tools.ide.url.updater.kotlin;
+
+import com.devonfw.tools.ide.url.folderhandling.UrlVersion;
+import com.devonfw.tools.ide.url.updater.OSType;
+import com.devonfw.tools.ide.url.updater.WebsiteCrawler;
+
+import java.util.regex.Pattern;
+
+public class KotlinNativeCrawler extends WebsiteCrawler {
+    @Override
+    protected String getToolName() {
+        return "kotlin";
+    }
+
+    @Override
+    protected String getEdition() {
+        return "kotlin_native";
```

Comment:
First I had it as kotlin_native, but when I implemented the 2 kotlin commandlets I had an issue when writing the kotlin-native commandlet, because I for example have a variable called KOTLIN_NATIVE_HOME currently.
I would have to call the variable KOTLIN-NATIVE_HOME tho to have the same naming accross the project but that lead to compiler issues.
![image](https://user-images.githubusercontent.com/102921542/228353507-8c6b9740-1b31-4c0a-9d0b-2eb47f8e257e.png)
Same goes for the KOTLIN_NATIVE_VERSION Variable that the user can manually set in devon. It would be KOTLIN-NATIVE_VERSION then. So to remove this redundancy I decided to leave it as kotlin_native...

Reply:
Thanks for the explanation. For KISS we can simply leave it as is for the merge.